Assets/Plugins/Editor/Uplift/DependencyResolution/DependencyGraph.cs

```diff
@@ -23,12 +23,14 @@
 // --- END LICENSE BLOCK ---
 
 using System.Collections.Generic;
+using System.Text;
+using Uplift.Common;
 using Uplift.Packages;
 using Uplift.Schemas;
 
 namespace Uplift.DependencyResolution
 {
-	class DependencyGraph
+	public class DependencyGraph
```

Comment:
Not necessary to change visibility if we make several of the classes non public.